initializr/src/main/resources/templates/home.html

```diff
@@ -111,13 +111,13 @@
                     </div>
                 </div>
             </div>
-            <div class="col-sm-6">
+            <div id="dependencies" class="col-sm-6">
                 <h3>Project dependencies</h3>
                 <% dependencies.each { %>
                 <div class="form-group col-xs-6">
                 <h4>${it.name}</h4>
                     <% it.content.each { %>
-                    <div class="checkbox">
+                    <div class="checkbox" data-range="1.2.1.RELEASE">
```

Comment:
This should be fixed with the real value coming from boot metadata. Leaving the value empty should be fine as well.